src/k8s/cmd/k8s/k8s_bootstrap.go

```diff
@@ -188,7 +188,8 @@ func getConfigInteractively(stdin io.Reader, stdout io.Writer, stderr io.Writer)
 	config.ServiceCIDR = vals.Pointer(serviceCIDR)
 
 	// TODO: any other configs we care about in the interactive bootstrap?
-
+	taints := askQuestion(stdin, stdout, stderr, "Any taints on the control nodes?", nil, "", nil)
+	config.ControlPlaneTaints = strings.FieldsFunc(taints, func(c rune) bool { return c == ',' })
 	return config
```

Comment:
does it make sense to ask such a question when bootstrapping interactively?

Reply:
if not, i'll just drop it